chtransmissionformat/README.md

```diff
@@ -0,0 +1,267 @@
+
+**Contact**
+
+Geschäftsstelle IG eMediplan<br>
+c/o Köhler, Stüdeli & Partner GmbH<br>
+Amthausgasse 18, 3011 Bern<br>
+Tel. +41 (0)31 560 00 24<br>
+info@emediplan.ch
+
+## Introduction
+
+Medication plans often need to be printed on paper.
+To easily bring these back into the digital world,
+the CHMED standard (in all its versions) requires a specification
+of how the whole information can be represented in a QR code
+as a simple line of text.
+
+The ChTransmissionFormat is this corresponding format
+which defines the contents of the QR code on a printed medication plan
+(which can be represented either physically on paper or digitally e.g. as a PDF).
+This representation can be used anywhere
+but is optimized for the usage in the QR code.
+
+### Goals
+
+- Define parsing rules based on prefix of the data
+  (only looking at the start of the data needs to be sufficient
+  to determine how the rest needs to be parsed),
+  specifically the version part
+  which allows to implement support for multiple versions.
+- Define measures to reduce the size of data since the amount of data
+  which can be included in a single QR code is limited.
+
+## Format
+
+A ChTransmissionFormat document must use UTF-8 encoding.
+
+```
+CHMED{ReleaseYear}{SubVersion}{AdditionalMetaData}{Data}
+```
+
+Representation as regex:
+
+```regex
+^CHMED([0-9]{2})([A-Z]+)(.*)$
+```
+
+Description and constraints:
+
+- `CHMED` (Prefix): This is the fixed string `CHMED`.
+  It is always the same value and should be used to determine if the given input string
+  should be tried to parse as ChTransmissionFormat.
+  During parsing, it should be treated as case-insensitive (e.g. `ChMed` should also be accepted)
+  but during serialization all uppercase should be used.
+- `ReleaseYear`: two digits (e.g. `16`)
+- `SubVersion`: At least a single uppercase alphabetic character (e.g. `A`).
+  Depending on the `ReleaseYear`, more than one character can be allowed.
+  During parsing, it should be treated as case-insensitive (e.g. `a` should also be accepted)
+  but during serialization all uppercase should be used.
+- `AdditionalMetaData`: Based on the `ReleaseYear`, additional meta data may be required.
+- `Data`: The actual data (JSON) which (after applying transformations based on the `ReleaseYear`)
+  needs to conform with the corresponding specification defining the format of the JSON.
+
+Since part of the format is dependent on the `ReleaseYear`,
+each available `ReleaseYear` is covered independently.
+Only years listed here are considered valid for use with ChTransmissionFormat.
+All constraints listed above also apply to the specific parts defined for a `ReleaseYear`.
+
+Note that the JSON format itself does not contain a version marker
+as it would be duplicated data when transmitted using ChTransmissionFormat
+and one of the goals is to reduce the data size.
+Therefore, CHMED documents should always be transmitted using ChTransmissionFormat.
```

Comment:
Is there an alternative? Kind of a weird statement in the specification of ChTransmissionFormat

Reply:
Yes, generally you can also transmit the JSON directly and somehow else provide what version is used (and potentially other header metadata). I tried to clarify this but maybe it would be better to move this to the CHMED16A/CHMED23A specification instead? What do you think?

Reply:
Ah I understand. I would either move it to the introduction or to the other files like you suggested

Reply:
Moved it to the main files of each specification 👍